dir.props

```diff
@@ -48,6 +48,12 @@
     <ToolsDir Condition="'$(UseToolRuntimeForToolsDir)'=='true'">$(ToolRuntimePath)</ToolsDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(ProjectDir)Tools/</ToolsDir>
     <ToolRuntimePath Condition="'$(ToolRuntimePath)'==''">$(ToolsDir)</ToolRuntimePath>
+    
+    <!-- Temporarily enable local build of tools -->
```

Comment:
Can we try to abstract any property or targets files into a common props/targets file under a common directory so it is easy to clean-up after we are done.